homeassistant/components/plum_lightpad/light.py

```diff
@@ -145,6 +185,16 @@ def name(self):
         """Return the name of the switch if any."""
         return self._name
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "name": self.name,
+            "identifiers": {(DOMAIN, self.unique_id)},
```

Comment:
Each Lightpad has a glow ring.  What exactly is `self_unique_id` supposed to be?  Maybe using the LightPad ID (lpid) in making the identifier?